sql/hive/src/main/scala/org/apache/spark/sql/hive/client/package.scala

```diff
@@ -62,6 +62,13 @@ package object client {
         "org.pentaho:pentaho-aggdesigner-algorithm",
         "net.hydromatic:linq4j",
         "net.hydromatic:quidem"))
+
+    case object v2_0 extends HiveVersion("2.0.1",
+      exclusions = Seq("eigenbase:eigenbase-properties",
```

Comment:
This was originally added because `VersionsSuite` would fail without it, since it uses mavin/ivy to download the different Hive libraries (because they used to depend on snapshot releases of libraries, or libraries that don't exist anymore on maven central). Perhaps Hive has updated its dependencies enough that some of this can be cleaned up...